docker/development/docker-compose.yml

```diff
@@ -1,6 +1,5 @@
 version: "3.4"
 x-shared-postgres-environment: &x-shared-postgres-environment
-  POSTGRES_DB: "${POSTGRES_DB:-mutualaid}"
```

Comment:
Don't use the same database for every environment. Defer to the application `database.yml` defaults instead.

Reply:
Would it be possible to set `POSTGRES_DB=''` in the docker-compose and Dockerfile files, and then change the behavior in the `database.yml` file so that it will use the default if `POSTGRES_DB` is an empty string? Then you keep the nice "inherit the expected value from the database.yml" behavior, but you can also override it with environment variables when building the image. I'm not sure if there's a standard for this in Ruby land - when I'm wearing my rather ill fitting & ad-hoc dev-ops hat, I like to do everything with environment variables. Otherwise I always seem to end up messing around with `sed` & friends trying to change some value in a configuration file somewhere.

Reply:
> Would it be possible to set POSTGRES_DB='' in the docker-compose and Dockerfile files, and then change the behavior in the database.yml file so that it will use the default if POSTGRES_DB is an empty string?

Aye, I currently believe that this could be accomplished using Ruby's [`Object#presence`](https://api.rubyonrails.org/classes/Object.html#method-i-presence) method in the ERB interpolation inside the `database.yml`.

Mayhaps something like the following:

```yml
development:
  ...
  database: <%= ENV['POSTGRES_DB'].presence || 'mutualaid_development' %>

test:
  ...
  database: <%= ENV['POSTGRES_DB'].presence || 'mutualaid_test' %>
```

Essentially, if a  non-nil, non-empty-string value is stored in the environment, use it - otherwise, use the default.

Reply:
Thoughts on https://github.com/rubyforgood/mutual-aid/pull/722/commits/df847ce0c48a36b4435f1eed2f73ffae6dc66b9f?